src/SlackConnector/Connections/Clients/File/FlurlFileClient.cs

```diff
@@ -45,5 +46,21 @@ public async Task PostFile(string slackKey, string channel, Stream stream, strin
             var response = JsonConvert.DeserializeObject<StandardResponse>(responseContent);
             _responseVerifier.VerifyResponse(response);
         }
+
+        public async Task DownloadFile(string slackKey, SlackFile file, string path)
```

Comment:
Please can you look into using Flurl to download a file rather than managing this yourself.

Reply:
I'll look into it

Reply:
Do you know the best practice for mocking a SlackFile? I was trying to use Mock, but the properties are read-only. I don't think giving it all the parameters the constructor needs would be a good solution

Reply:
There is a DownloadFile method you can use 😄 
```csharp
var path = await "http://files.foo.com/image.jpg"
    .DownloadFileAsync("c:\\downloads", filename);
```

Reply:
fixed, do you have a tip for the unit tests?